pkg/operator/ceph/controller/spec.go

```diff
@@ -75,22 +75,24 @@ var (
 	cronLogRotate = `
 CEPH_CLIENT_ID=%s
 PERIODICITY=%s
-LOG_ROTATE_CEPH_FILE=/etc/logrotate.d/ceph
-
-if [ -z "$PERIODICITY" ]; then
-	PERIODICITY=24h
-fi
+LOG_ROTATE_CEPH_FOLDER=/etc/logrotate.d/ceph
+DAEMON_LOG_FILE=/var/log/ceph/$CEPH_CLIENT_ID.log
```

Comment:
This isn't used anywhere right? What's the goal with this?

Reply:
removed